cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/data_explorer/discover.spec.js

```diff
@@ -133,14 +133,17 @@ describe('discover app', { scrollBehavior: false }, () => {
     });
 
     it('should reload the saved search with persisted query to show the initial hit count', function () {
+      // set current hit count as alias
+      cy.getElementByTestId('discoverQueryHits').invoke('text').as('hits');
       // apply query some changes
       cy.setTopNavQuery('test');
       cy.verifyHitCount('22');
 
       // reset to persisted state
       cy.getElementByTestId('resetSavedSearch').click();
-      const expectedHitCount = '14,004';
-      cy.verifyHitCount(expectedHitCount);
+      cy.get('@hits').then((hits) => {
+        cy.verifyHitCount(hits);
+      });
```

Comment:
i don't have context on this test case, but would this just verify that the actual number is the same as itself?

Reply:
actually i think i understand, we just want to make sure resetSavedSearch resets to the same number regardless of what it is

Reply:
yeah. We first grab the original hit count and save it for the alias `@hits`.

Then we modify the query to get a different hit count (22).

Then we click `resetSavedSearch`, and then verify that the hit count is back to the original (by comparing it with the alias `@hits`